core/domain/user_services.py

```diff
@@ -41,10 +41,10 @@
+    'data:image/webp;base64,UklGRj4HAABXRUJQVlA4WAoAAAAQAAAASwAASwAAQUxQSHQCAAABkN1s/5tW/6tIXDECLBCpc5cwQ6ZgCSQ2SPEagFP/UpAsym16L6eY9Ho6VtBf/M17Otj+DRARDtxGUqRk8RibnqCimZotzOuj9idjPrWP9HxhNqWSMldqGQZqWqVcAmSLl0B4Uqu6jT0/CPy9hlutnYTAZTEbL+OVEB61e8NAb1z9CGFlPD7GNHS9VcuQ2lWvC3osHtKOpbN2QSQv1jpYJx0Dc4/01n0i66/3eJyLyogDe01isbkHzkgkMjuYcp+Y7JcNO5kIjN5z7RGj3jX3o0Mz85WtW2L1douvM8OubzQDYjZo8jbcMvqVTUvs2k2+DnNl5p6mJQFtk/vM4GfdYSsgEYMtdkYG4XB9S0LeXuMM+pdhPBLTM/z/n5d+pEyClnlM//+yvX6S9Pf+e9GY7TVJ1GbPjv2LZp2EXUf/G73p+Enjdxj/mwprJO4alb/TU797kTwX3TD7hyIeAvQo/uGSVQmscqmUyvFoJWAfySlVYgMRakpKtXBl4NJSKRPeyOAmNKk8JwjxhHyBmhRqFOapSqHKvMaVgos+oiGFBkdt9qSwR/sTvhR8PhkCKQQYWcg6kXUTWQ8k6z8i678rK6aIinWzsmKwrNygSmgxOUtWLhWV42XVHrJqIlG1mqwaUlRtK6vmFtULyOpRRPVOono6Wb2mqB5YVG8uamYgapYhasYiavYjaiYlalYW3bRj6axFjPAXax2sk1axOKah660O/SO3q14X9JiKzfFKHx433CEqiRtXP0JYGVexmi1eAuFJreo29vwg8PcabrV2EgKXxayK31ypZRioaZVyKilT+cK8Pmp/MuZT+0jPF/IpFUkFVlA4IKQEAACQFQCdASpMAEwAPoU0l0elP7+hMfbfQ/AQiWwAv+PRCA/S8mD2hEpfjDAbZHzAedr6D/8z6gH9m6jX0AOlK8pnNLexn/P2RHaBW2yyNIvhR+ptnSem/YF3Uf9sTAGHNS0aeXf+ulMI+dqjc/YFZBg4c40l5QztZp+C3o0howNxL7QcZsgnXKnW3jlIgZ8WpwW8qbFyJ1TmDQ+f9+5+MHxz2dkeDPJE7kXZi8Z0qn+QiZOwy8wAAP75Ob2UMauDVOw4awyCdzHsQhVKVUMlZvb8KwvBNg1dAVL+/8yzj95LATUv+/bzXgG+1Fw7qeNYYogehrnbOJdNRmdK3vtW+79D+rl3DC4SK5yyvVgTb7vmlvnFChdWLLeeUclcPvBo7qaw5KHl3N6WN0Nf+Pg5B6NYsHTDWyqf4tlrTqnlPEDKbrX+fFvbS0SIEe3cfCdCjgtcopMvUQ9/IJElHP9uYeEVOstNSD6o2bkImN1oHaqPw+ofFl3hvwCbZTiI9T6aN2zwjCCwfeX0rjwEZhJv8bzHKdK2jdFCnsE47h5/0a+J/d22hYQS/JKhFbFYseD3BJcoDh+sQ/NyfDEC8ZJ9Nqe7WUiHkO4h69s8+bQFQpHdV6D2Ac4W2eff5HwDQMbK4fc389Z2zwiMrcO/Ae6PiZSam91/qjyi8ekxBlGProlsQLuo7ZMTVDn7efmAAMWb2ltyLuRQibgjUCaMxAC8tmFijfy8cvz1v2I/lZshls6ahWrHUOMHveYKKmZJu7mMWcxumDmr8DNhfpl9kJMrDbP2wIKmsVxN3+n40IPIKWm2tq13LkjU5f6QPlztNtQmV3zwJQjwNT0y9LSZbOrih3PrkXtHaij7rykvyzOKXpGLfDbw6yEo86dLiJ3Ad0SvrwWhNjmZ4XVjk8UCKs3yN3md9NcTGq9nRAhIWMbE+vx/OBuHh3BLDWBMvDMaXTR1DnH+2lif6w/RsKgwGQDnSXalw8NMFvPDgwdvY5PVium4+a1HWhvB7k96Xbftten82IqYW6d915cMMbEiISo2wjYo9x2hNeapikLyKe4JUDWBImCXQDWxiEWlJdtDJJe/E3AgtjaBtQS2sriMEGYSzyZ3C3716Fgrteg151kMYCm9TeDVRSBKXqU701i5KnS6gbKBqO/ih/4U01rQuBHrtf7GOSAe/zBlIlrMIMjH5wEf91J4K8N6qO6cyVsTbYcj78zx9m4aTpzTPzzoerbnR2UB9TgnLt/8flEzutLiZHau2TZxpqqIP5/F+tsRlrRRyXvdDkYezDShEkZc1KFq300JUumOE1k6J8a/SOm48/oVO4B3KJnyoSMWLQNNTQgy+WQdXXX1r/yFZn733jS99QugKc2oe8c9+tvCgX11vaWLmMuywC78f8pvk6tCal+hO6MPOWBonBG3IiAIXt2G3GbY8oCeGFq/OiaKdiACII6Y/Njyag8/dvf/5p//yz//+r17fHCKLXCXlseWCWAe64USMTxLK8yGWG0cX+IHQPGaa9foWMa5KuhaXvlHqJMlrSbAZ+f/Pc/uUBPH7dn7CXz0cs6CQXjRCQ0P2PqwzVdGmKAAAAA=')  # pylint: disable=line-too-long
```

Comment:
Thinking about it could we support both PNG and WebP here?

Reply:
Sure will update both the urls